{
  "bad": false,
  "new_version": "2021041502",
  "new_version_attempt_ts": {
    "0.2.24": 1758390788.0,
    "0.2.4": 1758387188.0,
    "2021041502": 1759542620
  },
  "new_version_attempts": {
    "0.1.11": 1,
    "0.1.14": 1,
    "0.2.13": 1,
    "0.2.17": 1,
    "0.2.18": 1,
    "0.2.24": 0,
    "0.2.4": 1,
    "0.2.8": 1,
    "2021041502": 23
  },
  "new_version_errors": {
    "0.2.24": "The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '0.2.24' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hexwatershed-{{ version }}.tar.gz'\n",
    "0.2.4": "The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '0.2.4' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hexwatershed-{{ version }}.tar.gz'\n",
    "2021041502": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/18237673321\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 233, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 270, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 311, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2021041502' to make sure they exist!\n\nWe also found the following errors:\n\n - migrations do not work on `git_url`s\n\n"
  }
}